index.bs

```diff
@@ -439,9 +477,13 @@ following steps:
 1. If the [=current settings object=]'s [=relevant global object=]'s
      [=associated Document=] is NOT [=fully active=], throw an {{InvalidStateError}}
      and abort these steps.
-1. If {{[[started]]}} is `true` and no <a event
-     for=SpeechRecognition>error</a> or <a event for=SpeechRecognition>end</a> event
-     have fired, throw an {{InvalidStateError}} and abort these steps.
+1. If {{SpeechRecognition/[[started]]}} is `true` and no <a event
+    for=SpeechRecognition>error</a> event or <a event for=SpeechRecognition>end</a> event
+    has fired on it, throw an {{InvalidStateError}} and abort these steps.
```

Comment:
We want to track this in a less hand-wavy way (flipping a bool when at least one has fired, and checking that).